backstop/tests/table.html

```diff
@@ -1867,1876 +1242,4740 @@ <h2 id="regular">Zebra striping</h2>
+                aria-label="More options"
+                type="button"
+              >
+                <!-- Use more.svg from @itwin/itwinui-icons package. -->
+                <svg
+                  class="iui-icon"
+                  aria-hidden="true"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    d="m4 8a2 2 0 1 1 -2-2 2 2 0 0 1 2 2zm4-2a2 2 0 1 0 2 2 2 2 0 0 0 -2-2zm6 0a2 2 0 1 0 2 2 2 2 0 0 0 -2-2z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+
+          <div class="iui-row iui-expanded">
+            <div class="iui-cell iui-slot">
+              <label class="iui-checkbox">
+                <input
+                  type="checkbox"
+                  id="select-all-checkboxes"
+                />
+                <span class="iui-checkbox-checkmark">
+                  <svg
+                    viewBox="0 0 16 16"
+                    aria-hidden="true"
+                    focusable="false"
+                  >
+                    <path
+                      class="iui-check-partial"
+                      d="m1 6.5h14v3h-14z"
+                    />
+                    <path
+                      class="iui-check"
+                      d="M6,14L0,8l2-2l4,4l8-8l2,2L6,14z"
+                    />
+                  </svg>
+                </span>
+              </label>
+            </div>
+            <div class="iui-cell iui-slot">
+              <button
+                class="iui-button iui-borderless iui-expander"
+                type="button"
+                aria-label="Expand row"
+              >
+                <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
+                <svg
+                  viewBox="0 0 16 16"
+                  class="iui-icon"
+                  aria-hidden="true"
+                >
+                  <path d="m4.7 0l-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
+                </svg>
+              </button>
+            </div>
+            <div class="iui-cell iui-main-column">Beta</div>
+            <div class="iui-cell">$12.00</div>
+            <div class="iui-cell iui-slot">
+              <button
+                class="iui-button iui-borderless iui-more-options"
+                aria-label="More options"
+                type="button"
+              >
+                <!-- Use more.svg from @itwin/itwinui-icons package. -->
+                <svg
+                  class="iui-icon"
+                  aria-hidden="true"
+                  viewBox="0 0 16 16"
+                >
+                  <path
+                    d="m4 8a2 2 0 1 1 -2-2 2 2 0 0 1 2 2zm4-2a2 2 0 1 0 2 2 2 2 0 0 0 -2-2zm6 0a2 2 0 1 0 2 2 2 2 0 0 0 -2-2z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+          <div class="iui-row iui-expanded-content">
```

Comment:
Only using the `.iui-expanded-content` class on the expandable rows that do not follow the table's column structure.